style.css

```diff
@@ -0,0 +1,112 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap");
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  background-color: #170606;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+header {
+  width: 375px;
+  height: 590px;
+  background: #1c1a19;
+  background-image: url(Illustration.png);
+  background-repeat: no-repeat;
+  background-size: 200px 300px;
+  background-position: 170px 320px;
```

Comment:
[Optional - but highly recommended]

In order for your app to be already responsive to different mobile screen sizes, you should consider removing the fixed positioning of the background image.

Again, please look at how the background image changes position when I am using different mobile screen sizes:
https://www.loom.com/share/488ec7addd2345dc921aa1db417daacc


- [ ] My suggestion is the following: `background-position: right bottom;`